classes/02class/exercises/c02-network06/gpavelar/README.md

```diff
@@ -0,0 +1,48 @@
+# c02-network06
+
+## Commands Execution Output
+
+- Commands for creating the NAT gateway:
+
+```bash
+## Allocate IP
+aws ec2 allocate-address --domain vpc
```

Comment:
for this exercise `domain` was *probably* not needed. Unless you have an old account that supports the deprecated "EC2-Classic" the domain will always be VPC.

No harm in setting it up though, I had to read the docs because I have never used it.